src/thor/timedep_forward.cc

```diff
@@ -49,13 +50,16 @@ void TimeDepForward::Clear() {
 // from the end node of any transition edge (so no transition edges are added
 // to the adjacency list or EdgeLabel list). Does not expand transition
 // edges if from_transition is false.
-bool TimeDepForward::ExpandForward(GraphReader& graphreader,
-                                   const GraphId& node,
-                                   EdgeLabel& pred,
-                                   const uint32_t pred_idx,
-                                   const TimeInfo& time_info,
-                                   const valhalla::Location& destination,
-                                   std::pair<int32_t, float>& best_path) {
+template <TimeDepForward::ExpansionType expansion_direction, typename EdgeLabelT>
```

Comment:
Note the EdgeLabelT - this is a temporary WIP trying to figure out how to handle EdgeLabel-vs-BDEdgeLabel.